tf2/include/tf2/LinearMath/Vector3.h

```diff
@@ -625,7 +625,7 @@ class tf2Vector4 : public Vector3
 TF2SIMD_FORCE_INLINE void	tf2SwapScalarEndian(const tf2Scalar& sourceVal, tf2Scalar& destVal)
 {
 	unsigned char* dest = (unsigned char*) &destVal;
-	unsigned char* src  = (unsigned char*) &sourceVal;
+	const unsigned char* src  = (const unsigned char*) &sourceVal;
```

Comment:
While you are changing this, I'll suggest you also change to a C++-style cast.  So the following:

```suggestion
	const unsigned char* src  = reinterpret_cast<const unsigned char*>(&sourceVal);
```